environment.py

```diff
@@ -230,9 +230,18 @@ def env() -> Mapping[str, Optional[str]]:
         #
         'AZUL_VERSIONED_BUCKET': 'edu-ucsc-gi-singlecell-azul-config-dev.{AWS_DEFAULT_REGION}',
 
-        # The number of workers pulling files from the repository (DSS or TDR).
+        # The number of workers pulling files from the DSS repository.
         # There is one such set of repository workers per index worker!
-        'AZUL_REPO_WORKERS': '8',
+        'AZUL_DSS_WORKERS': '8',
+
+        # The number of worker/s pulling files from the TDR repository.
+        # There is one such set of repository worker/s per index worker!
```

Comment:
Remove "/s". It makes no sense to describe the "number of worker", so there's no need for a separate grammar to cover  the case where the var is set to 1. It's also inconsistent with the documentation for the previous variable.